src/main/java/net/tlabs/tablesaw/parquet/TablesawParquetReader.java

```diff
@@ -33,32 +33,46 @@
 
 public class TablesawParquetReader implements DataReader<TablesawParquetReadOptions> {
 
-  private static final Logger LOG = LoggerFactory.getLogger(TablesawParquetReader.class);
+    private static final Logger LOG = LoggerFactory.getLogger(TablesawParquetReader.class);
 
-  @Override
-  public Table read(final Source source) throws IOException {
-    final File file = source.file();
-    if (file != null) {
-      return read(TablesawParquetReadOptions.builder(file).build());
+    /**
+     * Registers the TablesawParquetReader in the default tablesaw registry.
+     * Reader is associated with its options and the '.parquet' extension.
+     */
+    public static void register() {
```

Comment:
This static block should be in TablesawParquet and be private. If the idea is to have a static initializer/registration only for the reader (or writer) then TablesawParquet is not useful since it's only calling the reader and writer initializers.

As a user I would be confused on what I am supposed to use to "register" those reader/writers: use TablesawParquet.register() or use TablesawParquetReader(Writer).register()? Or do I need to call both? 

In addition, keeping those initializers in TablesawParquet lightens the code in the reader/writers and avoid having static/helper code for registration in multiple places. 

Last but not least, it's a matter of class concern/role: readers, writers read and write, TablesawParquet "entry point" registers what's necessary for Tablesaw - in contrast to current readers/writers reading and writing and registering.

Reply:
I agree. Changed in new commit.